packages/cli-platform-apple/src/config/findPodfilePath.ts

```diff
@@ -50,8 +51,13 @@ export default function findPodfilePath(
      */
     .sort((project) => (path.dirname(project) === platformName ? -1 : 1));
 
+  const supportedPlatformsArray: string[] = Object.values(supportedPlatforms);
+  const containsOnlySupportedPodfiles = podfiles.every((podfile) =>
+    supportedPlatformsArray.includes(podfile.split('/')[0]),
+  );
+
   if (podfiles.length > 0) {
-    if (podfiles.length > 1) {
+    if (podfiles.length > 1 && !containsOnlySupportedPodfiles) {
```

Comment:
Let’s rename to “containsUnsupportedPodfiles” to avoid negation in of statement

Reply:
Fixed 👍🏻